pkg/config/logconfig.go

```diff
@@ -16,17 +16,19 @@ import (
 type LogConfig interface {
 	SetLevel(level string)
 	GetMetricConfig() LogFileConfiguration
+	GetPublishedTransactionsConfig() LogFileConfiguration
 }
 
 // LogConfiguration -
 type LogConfiguration struct {
 	LogConfig
-	Level        string               `config:"level"`
-	Format       string               `config:"format"`
-	Output       string               `config:"output"`
-	File         LogFileConfiguration `config:"file"`
-	MetricFile   LogFileConfiguration `config:"metricfile"`
-	MaskedValues string               `config:"maskedvalues"`
+	Level                     string               `config:"level"`
+	Format                    string               `config:"format"`
+	Output                    string               `config:"output"`
+	File                      LogFileConfiguration `config:"file"`
+	MetricFile                LogFileConfiguration `config:"metricfile"`
+	PublishedTransactionsFile LogFileConfiguration `config:"publishedtransactionsfile"`
```

Comment:
Can we  usageFile (or something similar) to not confuse usage audit with transactions

Reply:
sure, will do.